src/service/jobRoleService.ts

```
@@ -2,15 +2,15 @@ import axios from "axios";
 import type { JobRole } from "../model/jobRole";
 
 export const jobRoleService = {
-    async getJobRoles(): Promise<JobRole[]> {
+    async getJobRoles(token?: string): Promise<JobRole[]> {
         try {
-            const response = await axios.get(process.env.API_URL + "api/job-roles");
+            const response = await axios.get(process.env.API_URL + "api/job-roles", {
+                headers: { Authorization: `Bearer ${token}` },
+            });
             if (response.status === 200) {
                 return response.data
             }
-        }
-        catch (e) {
-            throw new Error("Failed to get job roles")
-        }
+        } catch (e) { }
```

Did you mean to have an empty catch?

---

Yeah, otherwise I'd just duplicate line 14 inside the catch block and outside.